cpp/autosar/test/rules/M0-1-10.1/test.cpp

```diff
@@ -0,0 +1,34 @@
+#include <cstdint>
```

Comment:
Could you please add a non-compliant function into this test case? e.g. a function which is never called.

Reply:
Done in [0c20066](https://github.com/github/codeql-coding-standards/pull/647/commits/0c200662d4ccb8de83792676e9d2f3d0030d7cd2)